_data/navigation.yml

```diff
@@ -113,3 +136,21 @@ github-cli:
     url: /github-cli/15
   - title: "Merge Your PR"
     url: /github-cli/16
+
+advanced:
+  - title: "Welcome"
+    url: /reset/fc
+  - title: "Scenarios"
+    url: /reset/fc-01
+  - title: "Too Many (small) Commits"
+    url: /reset/fc-02
+  - title: "Commit Message Sucks"
+    url: /reset/fc-03
+  - title: "Committed to Wrong Branch"
+    url: /reset/fc-04
+  - title: "Accidental Commit"
+    url: /reset/fc-05
+  - title: "Commit Broke Everything!"
+    url: /reset/fc-06
+  - title: "Other Thing"
```

Comment:
Removed line 155, it was a leftover object from when I was mapping out the course.